src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -9,14 +12,19 @@
 @RequestMapping("/v1/task")
 public class TaskController {
 
+    @Autowired
+    private DbService service;
+    @Autowired
+    private TaskMapper taskMapper;
+
     @RequestMapping(method = RequestMethod.GET, value = "getTasks")
     public List<TaskDto> getTasks() {
-        return new ArrayList<>();
+        return taskMapper.mapToTaskDtoList(service.getAllTasks());
     }
 
     @RequestMapping(method = RequestMethod.GET, value = "getTask")
     public TaskDto getTask(Long taskId) {
-        return new TaskDto(1L, "test title", "test content");
+        return taskMapper.mapToTaskDto(service.getTaskById(2L));
```

Comment:
tutaj powinienes przekazywac do serwisu `taskId`, nie zahardcodowana 2L

Reply:
zahardcodowywałem cyfry żeby zobaczyć czy Postman działa i zapomniałem go podmienić
jeśli przekażę taskId to skąd będzie wzięte Id przez Postmana?

Reply:
pytanie nie ważne, źle wpisywałem adres